infra/app/app-config/env-config/variables.tf

```diff
@@ -11,6 +11,16 @@ variable "environment" {
   type        = string
 }
 
+variable "account_name" {
+  description = <<EOF
```

Comment:
Nit: I would suggest not using EOF because that implies "end of file" but something like EOT (which is used in the [terraform docs](https://developer.hashicorp.com/terraform/language/expressions/strings)) instead.